packages/patternfly-4/react-core/src/components/ChipGroup/Chip.js

```diff
@@ -93,7 +93,8 @@ Chip.defaultProps = {
   closeBtnAriaLabel: 'close',
   className: '',
   isOverflowChip: false,
-  tooltipPosition: 'top'
+  tooltipPosition: 'top',
+  onClick: () => {}
```

Comment:
Don't have to change this, but I've seen it written so many ways, like `null`, `() => undefined`, `Function.prototype` maybe we should have a noop util for consistency at some point